### PR TITLE
Add recipe for sphinxcontrib-plantuml

### PR DESCRIPTION
**Pull request by melund**
This recipe was created using grayskull. 

The recipe is needed as a dependency for `sphinx-needs` for which I am trying to create a conda-forge package. 

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/sphinxcontrib-plantuml/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
```
Comment (melund) [inline recipes/sphinxcontrib-plantuml/meta.yaml:22]:
```suggestion
    - python >=3.8
```

**Comment by melund** [comment]
@conda-forge/help-python , This is ready for a review.

File: recipes/sphinxcontrib-plantuml/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  sha256: 8fde531d92d1cfc2817fe3647b3f2d07e76682c4a84889c04a53e831f7c54432
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
```
Comment (ocefpaf) [inline recipes/sphinxcontrib-plantuml/meta.yaml:19]: Should be the as run.

```suggestion
    - python >=3.8
```

**Comment by melund** [comment]
@ocefpaf Thanks a lot. I added your suggestion.

**Pull request merged**